script.js

```diff
@@ -291,7 +268,6 @@ else{
 
 })
   
-
 let errorBox=document.querySelector('#errorMessage')
 let closeError=document.querySelector('#errorMessage button');
 closeError.addEventListener('click',function(e){
```

Comment:
errorBox, closeError move to top of the file

Reply:
done